sse2neon.h

```diff
@@ -815,6 +815,88 @@ FORCE_INLINE __m128 _mm_cmpneq_ps(__m128 a, __m128 b)
         vceqq_f32(vreinterpretq_f32_m128(a), vreinterpretq_f32_m128(b))));
 }
 
+// Compares for inequality.
+// https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/ekya8fh4(v=vs.100)
+FORCE_INLINE __m128 _mm_cmpneq_ss(__m128 a, __m128 b)
```

Comment:
It can be reimplemented as following:
```c
FORCE_INLINE __m128 _mm_cmpneq_ss(__m128 a, __m128 b)
{
    return _mm_cmplt_ps(a, b);
}
```

Reply:
But what if the value of `a[31:0]` is greater than `b[31:0]`?

Reply:
I'll keep the original implementation